src/server.js

```diff
@@ -13,17 +10,17 @@ const DB = process.env.DB_URL;
 
 // Implement mongoose
 mongoose
-  .connect(DB)
-  .then(() => console.log('Connected to MongoDB successfully'))
-  .catch((err) => console.log('Could not connect to MongoDB', err));
-
-app.use(express.urlencoded());
-app.use(express.json());
+  .connect(process.env.MONGODB_URI)
+  .then(() => {
+    
+    app.use(express.urlencoded());
+    app.use(express.json());
 
-userRoute(app);
-teamRoute(app);
+    userRoute(app);
+    teamRoute(app);
 
-// Start the server
-app.listen(PORT, () => {
-  console.log(`Server listening at http://localhost:${PORT}`);
-});
+    app.listen(PORT, () => {
+      console.log(`Server listening at http://localhost:${PORT}`);
+    });
+  })
```

Comment:
Finally we put it in the then ?

Reply:
yes, I followed a tutorial for implement jest correctly with mongoose and they were doing like this (but we can rollback if needed)